theano/sandbox/gpuarray/tests/test_nnet.py

```diff
@@ -209,11 +207,7 @@ def softmax_with_bias_unittest_template(dtypeInput, dtypeBias):
                       GpuSoftmaxWithBias)
 
     def cmp(n, m):
-        # print "test_softmax",n,m
-        if dtypeInput == 'float32':
-            data = numpy.arange(n * m, dtype='float32').reshape(n, m)
-        elif dtypeInput == 'float64':
-            data = numpy.arange(n * m, dtype='float64').reshape(n, m)
+        data = numpy.arange(n * m, dtype=dtypeInput).reshape(n, m)
 
         out = f(data)
         gout = f_gpu(data)
```

Comment:
test fails if run locally with wrong value and low error and nan.

On Thu, Sep 17, 2015 at 2:29 PM, abergeron <notifications@github.com> wrote:

> In theano/sandbox/gpuarray/tests/test_nnet.py
> <https://github.com/Theano/Theano/pull/3402#discussion_r39781486>:
>
> > @@ -209,11 +207,7 @@ def softmax_with_bias_unittest_template(dtypeInput, dtypeBias):
> >                        GpuSoftmaxWithBias)
> >
> >      def cmp(n, m):
> > -        # print "test_softmax",n,m
> > -        if dtypeInput == 'float32':
> > -            data = numpy.arange(n * m, dtype='float32').reshape(n, m)
> > -        elif dtypeInput == 'float64':
> > -            data = numpy.arange(n * m, dtype='float64').reshape(n, m)
> > +        data = numpy.arange(n * m, dtype=dtypeInput).reshape(n, m)
> >
> >          out = f(data)
> >          gout = f_gpu(data)
>
> Done.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Theano/Theano/pull/3402/files#r39781486>.
>